3459/1/link.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+require 'open-uri'
+require 'json'
+
+# This class reads the file and gets the names of the gems
+# and then gets their links to the github.
+class Link
```

Comment:
looks like it storing an array of links. I think, it will be better to make it with one static method `call` and use it and don't store any state. I think, it can be good for this class just to become link transformator from gem names to links.